bookserver/config.py

```diff
@@ -45,16 +45,16 @@ class Settings(BaseSettings):
     dev_dburl: str = "sqlite:///./runestone_dev.db"
     test_dburl: str = "sqlite:///./runestone_test.db"
 
-    # Configure ads.
+    # Configure ads. TODO: Link to the place in the Runestone Components where this is used.
     adsenseid: str = ""
     num_banners: int = 0
     serve_ad: bool = False
 
-    # :index:`docs to write`: **What's this?**
+    # TODO: where is this used? I don't see it in the code.
```

Comment:
library path was an early name for book path. It can be removed

Reply:
OK, done in my branch.